### DISCUSSION TITLE
日本旅游不完全踩坑记录

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#394] [Announcements]
# 日本旅游不完全踩坑记录

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2018/12/27/pit-stepping-record-for-japan/

**Comment by PsiACE** [comment]
DCC 什么的是真的寄，手续费高到离谱，上次刷卡被坑了好大一笔…（不过不是在日本）
